packages/webapp/src/utils/constants.js

```diff
@@ -171,6 +171,8 @@ const DEVELOPER_FULL_NAME = 'developer-fullname'
 const DEVELOPER_REFERER = 'developer-referer'
 const DEVELOPER_DETAILS_EMAIL = 'developer/details-email'
 const DEVELOPER_EMAIL_VALUE = 'developer-email-value'
+const DEVELOPER_DETAILS_EMAIL_CONFIRM = 'developer/details-email-confirm'
+const DEVELOPER_CONFIRM_EMAIL = 'developer-yes'
```

Comment:
This redis key needs a bit more description probably something like developer-confirm-email, developer-yes doesn't sound right?